doc/whats_new.rst

```diff
@@ -65,6 +65,8 @@ Changelog
 Bug
 ~~~
 
+- Fix one-sample baseline issue in :class:`mne.epochs.BaseEpochs` when using `tmin=0` by `Milan Rybar`_
```

Comment:
You'll need to add a URL for yourself at the bottom

Reply:
Also don't be afraid to enter your name as `Milan Rybář`if that's your preferred spelling — we're py3 only now, and hence unicode-friendly.